pkg/mimirpb/compat_test.go

```diff
@@ -463,7 +463,7 @@ func TestFromHistogramToHistogramProto(t *testing.T) {
 	p := FromHistogramToHistogramProto(ts, h)
 
 	expected := Histogram{
-		Count:          &Histogram_CountInt{18},
```

Comment:
this was necessary due to this change https://github.com/grafana/mimir-prometheus/commit/aa82fe198f66f4e2a41647fd0845834acf365e0e#diff-4c7063fbe57f049434ef7ed4d6360beb38a6cb4bd8154b2d5a2143557a5f52e3